specs/signing-scheme.md

```diff
@@ -48,35 +50,35 @@ This attribute dictates the rest of signature schema - the set of signed and uns
 
 Both `notary.x509` and `notary.x509.signingAuthority` signing schemes use the similar signature schema (set of signed and unsigned attributes) with the following differences.
 
-* `notary.x509` MUST use a countersignature from trusted source to determine authentic signing time (timestamp).
-This is supported through the Timestamp signature unsigned attribute in the signature envelope. Currently Notary Project uses a [RFC3161](ietf-rfc3161) compliant TSA signature for this purpose.
+* `notary.x509` can only use a timestamp countersignature to prove that the signature was generated before the time point been stamped by the `Timestamp Authority (TSA)`.
```

Comment:
nit: try not to use the term _only_. _can only use_ is interpreted as _is restricted to use_.
```suggestion
* `notary.x509` can use a timestamp countersignature to prove that the signature was generated before the time point been stamped by the *Timestamp Authority (TSA)*.
```